app/controllers/cohorts_controller.rb

```diff
@@ -31,8 +31,9 @@ def index
     @search = scope.ransack(params[:q])
 
     @cohort = cohort_source.new
-    @cohorts = @search.result.active.reorder(sort_string)
+    @cohorts = @search.result.active_user.reorder(sort_string)
     @inactive_cohorts = @search.result.inactive.reorder(sort_string)
+    @system_cohorts = @search.result.system_cohorts.reorder(sort_string)
```

Comment:
I _think_ we'll need to `@system_cohorts = @search.none` unless system cohorts is enabled.

Reply:
I think since the tab only shows if they are enabled, this isn't strictly needed -- but seems like a good idea...